lib/material/material_ui.flow

```diff
@@ -362,6 +362,13 @@ MAudioPlayer(filename : string, style : [MAudioPlayerStyle]) -> Material {
 	)
 }
 
+SampleStates(
```

Comment:
Why don't you use array of
SampleState(
	length : DynamicBehaviour<double>, 
	position : DynamicBehaviour<double>, 
	isPlay : DynamicBehaviour<bool>, 
	isReady : DynamicBehaviour<bool>
)?

Reply:
Because there are some part like fconnect(fsum(states.length), lenEx)